cmd/serve.go

```diff
@@ -419,35 +429,49 @@ func (c *ServeCommand) Command(serverService ServerServiceInterface, monitorServ
 		Run: func(cmd *cobra.Command, _ []string) {
 			ctx := cmd.Context()
 
-			// Setup default Crash Tracker client
+			// Setup the Crash Tracker client
 			crashTrackerClient, err := di.NewCrashTracker(ctx, crashTrackerOptions)
 			if err != nil {
 				log.Ctx(ctx).Fatalf("error creating crash tracker client: %s", err.Error())
 			}
 			serveOpts.CrashTrackerClient = crashTrackerClient
 
-			// Setup default Email client
+			// Setup the Email client
 			emailMessengerClient, err := di.NewEmailClient(emailOpts)
 			if err != nil {
 				log.Ctx(ctx).Fatalf("error creating email client: %s", err.Error())
 			}
 			serveOpts.EmailMessengerClient = emailMessengerClient
 			adminServeOpts.EmailMessengerClient = emailMessengerClient
 
-			// Setup default SMS client
-			smsMessengerClient, err := di.NewSMSClient(smsOpts)
+			// Setup the SMS client
+			serveOpts.SMSMessengerClient, err = di.NewSMSClient(smsOpts)
 			if err != nil {
 				log.Ctx(ctx).Fatalf("error creating SMS client: %s", err.Error())
 			}
-			serveOpts.SMSMessengerClient = smsMessengerClient
 
-			// Setup default AP Auth enforcer
+			// Setup the AP Auth enforcer
 			apAPIService, err := di.NewAnchorPlatformAPIService(serveOpts.AnchorPlatformBasePlatformURL, serveOpts.AnchorPlatformOutgoingJWTSecret)
 			if err != nil {
 				log.Ctx(ctx).Fatalf("error creating Anchor Platform API Service: %v", err)
 			}
 			serveOpts.AnchorPlatformAPIService = apAPIService
 
+			// Setup the TSSDBConnectionPool
+			tssDBConnectionPool, err := di.NewTSSDBConnectionPool(ctx, di.TSSDBConnectionPoolOptions{DatabaseURL: globalOptions.DatabaseURL})
+			if err != nil {
+				log.Ctx(ctx).Fatalf("error getting TSS DB connection pool: %v", err)
+			}
```

Comment:
Can we call `defer tssDBConnectionPool.Close()` here?

Reply:
Done in https://github.com/stellar/stellar-disbursement-platform-backend/pull/158/commits/7c6d5c09156f6f59c9fc68bbf5d7841b286e5dbd.